IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Documentation.md

```diff
@@ -7,7 +7,7 @@ The _IfcMeasureResource_ schema specifies units and defined measure types that m
 In different types of units there are five basic cases:
 
 * Basic SI-units, which cover a number of fundamental units of mainly physical quantities defined by [ISO-1000](../../bibliography.htm#iso-1000){ .int-ref} such as meter or millimeter as unit for length measure or square meter as a unit for area measure. The unit may have a scaling prefix (for example: milli, kilo).
-* Derived SI-units, which are defined as a derivation of the basic SI-units, for example, Newton (kg m / s2) as a unit of force. Both basic and derived SI-units are in the IFCs represented by IfcSIUnit.
+* Derived SI-units, which are defined as a derivation of the basic SI-units, for example, Newton (kg m / s2) as a unit of force. Both basic and derived SI-units are in the IFCs represented by IfcSIUnit
```

Comment:
```suggestion
* Derived SI-units, which are defined as a derivation of the basic SI-units, for example, Newton (kg m / s2) as a unit of force. Both basic and derived SI-units are in the IFCs represented by IfcSIUnit.
```

I don't understand the reason for this change. All other bullet points end with a `.`.